### PR TITLE
iTop Updates

### PR DESCRIPTION
**Pull request by bborofka**
Appending README documentation and updating scripts to use ArgumentParser() over global variables.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -367,3 +367,123 @@ required arguments:
...
+#### Configure LogicMonitor
+In your LogicMonitor portal, identify the Devices that you want to federate to iTop. Create a custom property `itop.class` and give it the value `Server`, `Printer` or `Hypervisor`. Apply the property at a Device or Device Group level. This will be used to federate devices from LogicMonitor to iTop with the appropriate CI class.
+
+You will also need to identify the Collector description that you want to use, if synchronizing devices from iTop to LogicMonitor.
+
+#### Execute lm_itop_sync.py Script (LogicMonitor to iTop)
+**lm_itop_sync.py** script can be executed on its own schedule, with the following arguments (gathered from iTop Configuration above):
+
+```
+$> python lm_itop_sync.py -h
```
Comment (woz5999) [inline README.md]: Update path to execute from example directory: > python ./examples/...

File: README.md [hunk 2]
```diff
@@ -367,3 +367,123 @@ required arguments:
...
+  -st SERVER_TABLE      iTop MySQL staging table name for Server class
+  -pt PRINTER_TABLE     iTop MySQL staging table name for Printer class
+  -ht HYPERVISOR_TABLE  iTop MySQL staging table name for Hypervisor class
+```
+
+#### Execute itop_lm_sync.py Script (iTop to LogicMonitor)
+The **itop_lm_sync.py** script can be executed on its own schedule, with the following arguments (gathered from iTop and LogicMonitor Configurations above):
+
+```
+$> python itop_lm_sync.py -h
```
Comment (woz5999) [inline README.md]: Update path to execute from example directory: > python ./examples/...

File: README.md [hunk 3]
```diff
@@ -367,3 +367,123 @@ required arguments:
...
+  -mh MYSQL_HOST        iTop MySQL server hostname (optional)
+  -mu MYSQL_USERNAME    iTop MySQL server username
+  -mp MYSQL_PASSWORD    iTop MySQL server password
+  -md MYSQL_DATABASE    iTop MySQL server database name
+  -st SERVER_TABLE      iTop MySQL staging table name for Server class
+  -pt PRINTER_TABLE     iTop MySQL staging table name for Printer class
+  -ht HYPERVISOR_TABLE  iTop MySQL staging table name for Hypervisor class
+```
+
+#### Execute itop_lm_sync.py Script (iTop to LogicMonitor)
```
Comment (woz5999) [inline README.md]: Match style conventions in rest of document. ### (3, not 4)

File: README.md [hunk 4]
```diff
@@ -367,3 +367,123 @@ required arguments:
...
+Lastly, you'll need to get the **Organization ID** you want to create the devices under, in iTop. Navigate to *Data administration > Organizations* and get the decide if you want to use one of the three defaults (*Demo*, *IT Department* or *My Company/Department*) or create a new one.
+
+Once decided, hover your cursor over the organization name and note the # of the organization id, it will be used as an argument with the **lm_itop_sync.py** script.
+
+#### Configure LogicMonitor
+In your LogicMonitor portal, identify the Devices that you want to federate to iTop. Create a custom property `itop.class` and give it the value `Server`, `Printer` or `Hypervisor`. Apply the property at a Device or Device Group level. This will be used to federate devices from LogicMonitor to iTop with the appropriate CI class.
+
+You will also need to identify the Collector description that you want to use, if synchronizing devices from iTop to LogicMonitor.
+
+#### Execute lm_itop_sync.py Script (LogicMonitor to iTop)
```
Comment (woz5999) [inline README.md]: Match style conventions in rest of document. ### (3, not 4)

**Comment by woz5999** [comment]
The code changes look good.

I hate to do this, but after seeing the extent of the instructions necessary for your scripts, I think it makes sense to create an examples subdirectory for your code with its own README. Can you modify your branch with the following:

create a directory named iTop-sync in the examples folder
move your script files to ./examples/iTop-sync/
remove your changes to ./README.md
create a new README.md in ./examples/iTop-sync/

**Comment by bborofka** [comment]
Changes have been made.

**Comment by bborofka** [comment]
Correct README committed.

**Pull request merged**